index.html

```diff
@@ -94,11 +94,151 @@ <h2>Software Developer</h2>
           <li></li>
         </ul>
 
-        <div class="work">
+        <div class="work-direction">
           <img src="/images/Arrow.png" alt="down-arrow" />
           <span>Work</span>
         </div>
       </div>
+
+      <!-- Work -->
```

Comment:
[OPTIONAL] Please do not leave unused code as a comment in your codebase. Unused code as comments makes your code not clean.
**Kindly remove all commented code in your index.html file 😊 .**